src/app/public/modules/fluid-grid/fluid-grid.component.scss

```diff
@@ -1,5 +1,29 @@
 @import "scss/variables";
 
-.sky-fluid-grid {
-  padding: 0 $sky-fluid-grid-gutter;
+@mixin create_columns($gutter-size) {
+  .sky-fluid-grid-gutter-size-#{$gutter-size} {
+    padding-left: map-get($gutter-sizes, $gutter-size);
+    padding-right: map-get($gutter-sizes, $gutter-size);
+
+    ::ng-deep .sky-row {
```

Comment:
The mixins are very nice! What do you think about using `:host-context` instead of the deprecated `:ng-deep`? I think it will isolate the styles to their respective stylesheets better: https://github.com/blackbaud/skyux-layout/pull/66